src/test/shared/filesystemUtilities.test.ts

```diff
@@ -29,6 +29,35 @@ describe('filesystemUtilities', () => {
         await del([tempFolder], { force: true })
     })
 
+    describe('mkdtemp', () => {
+        it(`makes temp dirs under filesystemUtilities.tempDirPath ('${tempDirPath}')`, async () => {
+            assert(
+                tempFolder.indexOf(tempDirPath) === 0,
+                `expected tempFolder ('${tempFolder}') to be in tempDirPath ('${tempDirPath}')`
+            )
+            const tmpDirExists = await fileExists(tempFolder)
+            assert(
+                tmpDirExists,
+                `tempFolder should exist: '${tempFolder}'`
+            )
+        })
+    })
+
+    describe('mkdtemp nested', () => {
```

Comment:
This still describes `mkdtemp`, so it should be merged into the above `describe` block.

Reply:
✔️